terraform/modules/erouska/coviddata.tf

```diff
@@ -10,15 +10,41 @@ locals {
     "roles/datastore.user"
   ]
 
-  calculatecoviddata_roles = [
+  getcoviddata_roles = [
+    "roles/cloudfunctions.serviceAgent",
+    "roles/datastore.viewer"
+  ]
+
+  changepushtoken_roles = [
     "roles/cloudfunctions.serviceAgent",
     "roles/datastore.user"
   ]
-  getcoviddata_roles = [
```

Comment:
why is this deleted? GetCovidData is still there...

Reply:
is it deleted? https://github.com/covid19cz/erouska-backend/blob/sv-iam-service-accounts/terraform/modules/erouska/coviddata.tf#L13

Reply:
ahaa! sorry, just the github diff didn't manage to pick up the change!